src/Command/AbstractCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 project - inspiring people to share!
+ * (c) 2020-2023 Oliver Bartsch, Benni Mack & Elias Häußler
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace TYPO3\Tailor\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use TYPO3\Tailor\Environment\Variables;
+use TYPO3\Tailor\Exception\ExtensionKeyMissingException;
+use TYPO3\Tailor\Filesystem\ComposerReader;
+
+/**
+ * Abstract class for tailor console commands.
+ */
+abstract class AbstractCommand extends Command
```

Comment:
Not sure about this additional inheritance layer. Usually an abstract class should at least define one abstract method.

Reply:
Yeah, I agree that this is not a nice coding style. Actually, it'd be enough to extract the `getExtensionKey()` to a helper class or similar. I'll prepare something.

Reply:
Fixed with a54b50e.